reporting-guide.md

```diff
@@ -0,0 +1,74 @@
+*   Any extra context you believe existed for the incident.
+*    If you believe this incident is ongoing.
+*    Any other information you believe we should have.
+
+### What happens after you file a report?
+
+The working group will immediately meet to review the incident and determine:
+
+*    What happened.
+*    Whether this event constitutes a code of conduct violation.
+*    Who the bad actor was.
+*    Whether this is an ongoing situation, or if there is a threat to anyone's physical safety.
+
+Conference staff will be happy to help participants contact venue security or local law enforcement, provide escorts, or otherwise assist those experiencing harassment to feel safe for the duration of the conference. We value your attendance.
+
+If this is determined to be an ongoing incident or a threat to physical safety, the staff's immediate priority will be to protect everyone involved. This means we may delay an "official" response until we believe that the situation has ended and that everyone is physically safe.
+
+Once the working group has a complete account of the events they will make a decision as to how to response. Responses may include:
+
+ *   Nothing (if we determine no violation occurred).
+ *   A private reprimand from the working group to the individual(s) involved.
+ *   A public reprimand.
+ *   Expulsion from the conference with no refund.
+ *   A permanent or temporary ban from some or all PyCon India spaces (mailing lists, IRC, etc.)
+ *   A request for a public or private apology.
+
+We'll respond within one week to the person who filed the report with either a resolution or an explanation of why the situation is not yet resolved.
+
+Once we've determined our final action, we'll contact the original reporter to let them know what action (if any) we'll be taking. We'll take into account feedback from the reporter on the appropriateness of our response, but we don't guarantee we'll act on it.
+
+Finally, the PyCon India Staff will make a report on the situation.
+
+What if your report concerns a possible violation by a staff member?
+
+
+In that case, you can make a report directly to any or all of the current (vice/co) chairs of the Code of Conduct committee. Their e-mail addresses are listed on the Code of Conduct Committee page. The chairs will follow the usual enforcement process with the other members, but will exclude the member(s) that the report concerns from any discussion or decision making.
+
+If your report concerns all current (vice/co) chairs of the committee, please send your report directly to the PSF Board of Directors at  psf@python.org instead.
+
+Thank you for helping to make PyCon India a welcoming, friendly event for all.
+
+## Contact Information
+
+If the matter is especially urgent, you may also ask to be put in touch with the conference chair - Vijay Kumar
```

Comment:
This has to be changed. Sharmila please use your name over here. Also we will need to add contact information, like mobile no. and email. Please let me know if you would be willing to disclose your mobile no. / email over here.